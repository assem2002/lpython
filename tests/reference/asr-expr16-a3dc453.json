{
    "basename": "asr-expr16-a3dc453",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/expr16.py",
    "infile_hash": "c87ce02d4b227d695bad17d79c7359c0d8ffd64b5c5c231bce5528e4",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-expr16-a3dc453.stderr",
    "stderr_hash": "5e2be92ee502526a6c5f1aa78d0165103f5f066c76918ee8fcd3bcb1",
    "returncode": 2
}