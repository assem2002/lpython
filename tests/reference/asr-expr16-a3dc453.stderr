semantic error: Type mismatch in shorthand operator, the types must be compatible
 --> tests/expr16.py:6:9
  |
6 |         sum += data[i]
  |         ^^^    ^^^^^^^ type mismatch ('f64' and 'i32')


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
